Source/Core/OrientedBoundingBox.js

```diff
@@ -66,7 +66,7 @@ define([
         /**
          * The transformation matrix, to rotate the box to the right position.
          * @type {Matrix3}
-         * @default {@link Matrix3.IDENTITY}
+         * @default {@link Matrix3.ZERO}
          */
         this.halfAxes = Matrix3.clone(defaultValue(halfAxes, Matrix3.ZERO));
```

Comment:
I think the changes here should be reversed. The doc is correct, but `Matrix3.ZERO` should be `Matrix3.IDENTITY`.

`2x2x2` is correct because the half axes are `1x1x1` but the full cube is `2x2x2`.